### DISCUSSION TITLE
Add tooltip to ft.Tab()

### DISCUSSION THREAD

**Discussion by vitaluska123** [flet-dev/flet#1539] [Ideas]
For example, I have tabs in the application that have only an icon and so that the user can understand why the add **tooltip** tab

**Comment by vindevoy** [comment]
I support the idea.  I have one last tab, with the "plus" sign.  When I click on that tab, I add a tab, as you may guess.  I would like a tooltip on that tab saying "Add a data tab"

**Comment by FeodorFitsner** [comment]
Why not to use https://flet.dev/docs/controls/tooltip control?

**Comment by vindevoy** [comment]
I have a couple of reasons:

- First of all, I did not think of it, as all other controls seem to have this as a standard property, so I would consider this plan B
- My tabs look like this "Data 1" - " + ".  When you click the " + " tab, I add a tab between the "Data 1" and the " + ", called "Data 2".  I would have to move the tooltip accordingly.  Which is still possible, I guess.
- However, I can rename the tab.  Then I'm not sure where to position the tooltip.

But basically, it's available on all other controls, so for consistency, I would add it also on the tab.
